### PR TITLE
edit resnet50 benchmark_logs

### PR DESCRIPTION
**Pull request by danleifeng**
Add resnet50 benchmark_logs for validation.

### CODE REVIEW

File: benchmark/collective/resnet/train_with_fleet.py [hunk 1]
```diff
@@ -553,18 +570,15 @@ def if_exist(var):
...
-                    pass_id, "%.5f"%train_loss, "%.5f"%train_acc1, "%.5f"%train_acc5, "%.2f" % train_speed))
-
-        sys.stdout.flush()
+                        result['0']['acc1'] = test_acc1
+                        result['0']['acc5'] = test_acc5
+                        result['0']['result_log'] = dict()
+                        result['0']['result_log']['acc1'] = acc1_logs
+                        result['0']['result_log']['acc5'] = acc5_logs
+                        # max speed of all epochs
+                        result['1'] = max(train_speed_list) * num_trainers
```
Comment (gongweibao) [inline benchmark/collective/resnet/train_with_fleet.py]: median

File: benchmark/collective/resnet/train_with_fleet.py [hunk 2]
```diff
@@ -508,40 +512,53 @@ def if_exist(var):
             train_acc5 = np.array(train_info[2]).mean()
         train_end=time.time()
         train_speed = (batch_id * train_batch_size) / (train_end - train_begin)
+        train_speed_list.append(train_speed)
+
+        test_py_reader.start()
```
Comment (gongweibao) [inline benchmark/collective/resnet/train_with_fleet.py]: if args.validate:

**Review by gongweibao — CHANGES_REQUESTED** [review]

File: benchmark/collective/resnet/train_with_fleet.py (hunk 1 above)
Comment (danleifeng) [reply benchmark/collective/resnet/train_with_fleet.py]: done.

File: benchmark/collective/resnet/train_with_fleet.py (hunk 2 above)
Comment (danleifeng) [reply benchmark/collective/resnet/train_with_fleet.py]: done.

**Review by gongweibao — APPROVED** [review]

**Pull request merged**